packages/devtools_app/lib/src/shared/diagnostics/variable_factory.dart

```diff
@@ -344,8 +365,25 @@ List<DartObjectNode> createVariablesForAssociations(
     false,
     (p, e) => p || isPrimativeInstanceKind(e.key.kind),
   );
-  for (var i = 0; i < associations.length; i++) {
+  for (final i in Iterable.generate(associations.length)) {
     final association = associations[i];
+
+    if (asReferences) {
```

Comment:
will we eventually have to differentiate between whether this should be added as a live reference or a static reference?

Reply:
Static references never show up here. I plan to reorganize code to make it more supportable.